content/derisking-government-tech/resources/open-source-security.md

```diff
@@ -3,7 +3,77 @@ title: Best practices for open source software security
 permalink: /derisking-government-tech/resources/open-source-security/
 layout: layouts/page
 tags: derisking-government-tech
-sidenav: false
+description: High-level explanation for keeping data, static assets, secrets, and code safe in an open source project.
+eleventyNavigation:
+  parent: derisking-government-tech-resources-open-source-security
+  key: open-source-security
+  order: 1
+  title: Best practices for open source software security
```

Comment:
I think we should maybe shorten this to "Open source security" since it ends up wrapping onto three lines:

![image](https://github.com/user-attachments/assets/5033528a-3b37-4075-bd6c-0f6906330b03)

Reply:
Huh it actually looks like in the Resources section this behavior is _only_ working on the open source security page

Reply:
Sorry I missed the bugs in the side-nav behavior. The issue on the "Market research" and "Verbal interview questions" pages is that I used `h3`s rather than `h2`s to match the subheading font style in the design. Semantically, I should use `h2`s.

I'll make that change; if needed I can update the h2 style for those specific pages, although I'd like to try it out without that step first.

Reply:
I pushed up the h3-to-h2 change for those pages and updated the PR description accordingly.